client/signup/steps/domains/index.jsx

```diff
@@ -21,7 +21,8 @@ var StepWrapper = require( 'signup/step-wrapper' ),
 	{ getCurrentUser, currentUserHasFlag } = require( 'state/current-user/selectors' ),
 	{ DOMAINS_WITH_PLANS_ONLY } = require( 'state/current-user/constants' ),
 	analyticsMixin = require( 'lib/mixins/analytics' ),
-	signupUtils = require( 'signup/utils' );
+	signupUtils = require( 'signup/utils' ),
+	Abtest = require( 'lib/abtest' ).abtest;
```

Comment:
For consistency with other places where we import a function, I'd make `Abtest` all lowercase `abtest`.